pkg/apis/v1alpha1/types.go

```diff
@@ -62,6 +62,8 @@ type ConsulSpec struct {
 	// pod. If not specified, the pod will attempt to use a local agent on
 	// the host on which it is running.
 	Address string `json:"address,omitempty"`
+	// The location of a secret to mount with the Consul root CA.
+	CertificateAuthoritySecret string `json:"caSecret,omitempty"`
```

Comment:
this can actually be removed from both this and the generated CRD, it should have been removed as part of https://github.com/hashicorp/consul-api-gateway/pull/72, but looks like we forgot to regenerate the CRDs after.

Reply:
Also, looks like the demo configs/fixture files will need to be updated accordingly -- but yeah, this isn't used in the codebase anymore to avoid the oddities of having to copy around secrets to different namespaces b/c of Kubernetes secret mounting constraints (you can only mount a secret if it exists in the same namespace as a pod).